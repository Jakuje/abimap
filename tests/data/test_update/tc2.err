[WARNING] Overwriting existing file 'tc2.map'
[WARNING] Given paths in '--out' and 'file' are the same.
[WARNING] Moving 'tc2.map' to 'tc2.map.old'.
[WARNING] ABI break detected: symbols were removed.
