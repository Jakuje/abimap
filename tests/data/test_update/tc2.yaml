# Testcases to test new command
-
  input:
    args:
      - "update"
      - "--remove"
      - "--out"
      - "tc2.map"
      - "tc2.map"
    stdin: "tc2.in"
  output:
    file: "tc2.out"
    stdout: "tc2.stdout"
    stderr: "tc2.err"
