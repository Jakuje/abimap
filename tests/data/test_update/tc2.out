# This map file was automatically updated

LIBTC2_2_0_0
{
    global:
        another_symbol;
    local:
        *;
} ;

